### PR TITLE
New details

### LINKED ISSUE #134: List view & header improvements
This one I'm not sure and it's not a priority at all, however I spent some time on preparing more visually appealing and with better visual hierarchy look for widget details.

Significant changes:
- Menu to keep the structure and show that it's all about sub-groups
- Show the wallet icon with address of widget's owner
- Swap the 'Contact' button with 'Dashboard' button, if detected owner
- Visuals 

betterlook

### PR DESCRIPTION
**Pull request by jakubsta**
Resolve #134

### CODE REVIEW

File: apps/main/src/pages/Direct/Details/index.tsx [hunk 1]
```diff
@@ -28,29 +29,36 @@ class Details extends Component<IProps, IState> {
...
+            openWidgetDetails={this.closeModal}
           />
         </Modal>
         <Status asset={widgetSettings.asset} />
       </div>
     );
   }
 
-  _onAddLink = () => {
+  private onAddLink = () => {
```
Comment (michalwarda) [inline apps/main/src/pages/Direct/Details/index.tsx]: :+1:

File: packages/new-details/components/Expires.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from 'styled-components';
+import moment from 'moment';
+
+import { SmallBlackBoldText } from './Text';
+
+const getColor = (expiresIn) => {
+  if (expiresIn < moment.duration({ weeks: 1 })) {
```
Comment (michalwarda) [inline packages/new-details/components/Expires.tsx]: `const getColor = (expiresIn) => expiresIn < moment.duration({ weeks: 1 }) ?  '#fb0035' : '#09d57c';

File: packages/new-details/components/List.tsx [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+import { ITokenDetails } from '@linkexchange/token-details-provider';
+import { mobileOrTablet } from '@linkexchange/utils/userAgent';
+import BoostArrow from '@linkexchange/images/arrow-boost.svg';
+
+import LinksStore from '../linksStore';
+import { Columns, Column, FlexColumn } from './Columns';
+import { BlackBoldText, BlueBoldText, LightGreyText } from './Text';
+import Hr from './Hr';
+
+const SmallGreenText = styled.p`
```
Comment (michalwarda) [inline packages/new-details/components/List.tsx]: maybe use span and remove inline

File: packages/new-details/components/List.tsx [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+  align-items: center;
+  justify-content: ${(props) => (props.justifyContent ? props.justifyContent : '')};
+`;
+
+const LinkTitle = BlackBoldText.extend`
+  min-width: 250px;
+  width: 250px;
+`;
+
+const LinkTarget = styledComponentWithProps<{}, HTMLLinkElement>(BlueBoldText.extend)`
```
Comment (michalwarda) [inline packages/new-details/components/List.tsx]: :+1:

File: packages/new-details/components/List.tsx [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+const TokenAmount = styled.span`
+  padding-top: 10px;
+  color: #acb7f5;
+  font-size: 12px;
+  font-weight: bold;
+  white-space: nowrap;
+`;
+
+const bounce = keyframes`
+	0%, 20%, 50%, 80%, 100% {
```
Comment (michalwarda) [inline packages/new-details/components/List.tsx]: fix indentation

File: packages/new-details/utils/index.ts [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { StyledFunction } from 'styled-components';
+
+export function styledComponentWithProps<T, U extends HTMLElement = HTMLElement>(
```
Comment (michalwarda) [inline packages/new-details/utils/index.ts]: najs

Comment (jakubsta) [reply packages/new-details/utils/index.ts]: "borrowed" from https://github.com/styled-components/styled-components/issues/630

**Pull request merged**